web/cobrands/sass/_base.scss

```diff
@@ -1311,7 +1311,7 @@ input.final-submit {
   margin-bottom: 0.5em;
 }
 
-%item-list__item--link {
+%item-list {
   padding: 0;
```

Comment:
This padding does nothing when extended by wards, because `.item-list` already has padding:0; this is a list item needed thing.